package.json

```diff
@@ -39,17 +41,22 @@
     "@typescript-eslint/eslint-plugin": "4.31.2",
     "@typescript-eslint/parser": "4.31.2",
     "axios": "^0.27.2",
+    "chai": "^4.3.7",
     "commitizen": "^4.2.4",
+    "dotenv": "^16.0.3",
     "eslint": "7.32.0",
     "eslint-config-airbnb": "18.2.1",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-react": "^7.26.0",
+    "ethereum-waffle": "^3.4.4",
+    "hardhat": "^2.12.2",
     "husky": "^7.0.2",
     "jest": "^27.2.2",
     "lint-staged": "11.1.2",
+    "mocha": "^10.1.0",
```

Comment:
having 2 different test runners might create big conflicts long term. Why did we add "mocha" when the project has "jest" configured. Is jest not used? This is fishy as no more than 1 test runner should be required per project

Reply:
We are deprecating jest from now on. So perhaps I should also remove it from the package.json. The reason we went with mocha is that mocha is compatible with hardhat which allows us to fork mainnet and use that as our testing environment instead of running a node locally. This will also allow us to run the tests in github ci.

Reply:
well lets remove jest and jest related deps from project. No need to keep libraries you don't use. WDYT?

Reply:
Yes, I agree with removing jest from the project.

Reply:
Removed

Reply:
can you share the commit where you did this @Sol3iman ?

Reply:
[abbf1bc](https://github.com/Voltz-Protocol/v1-sdk/pull/88/commits/abbf1bcad0f03813652a0268d09d26a256bd6cbc) and [f5a1475](https://github.com/Voltz-Protocol/v1-sdk/pull/88/commits/f5a14754f938eda976b65026766ab444a7b45fd2) as I forgot the jest.config.ts files